docs/admin/Offline_System_Certificate_Renewal.md

```diff
@@ -6,71 +6,105 @@ Offline System Certificate Renewal
 PKI server provides a mechanism to recover from expired system certificates. This mechanism can also be
 used to renew the certificates before they expire. There are 2 ways to renew the certs
 
-1. [Automated Renewal Process](#Automated-Renewal-Process)
-2. [Manual Renewal Process](#Manual-Renewal-Process) 
+1. [Automated Renewal Process](#Automated-Renewal-Process) - supports LDAPS/LDAPI configuration
+2. [Manual Renewal Process](#Manual-Renewal-Process) - supports LDAP/LDAPS/LDAPI configuration
 
-It is assumed that you have the following:
-* Valid CA signing cert
-* Valid admin cert
+This tool's behavior is different in an **IPA environment** and **standalone PKI environment**
 
-To verify these assumptions are valid:
+**NOTE:** If you have a **non-secure** LDAP setup and if you **don't know the agent username/password**, you cannot use this tool
 
-1. List details of all system certificates. (Note down the `<cert ID>` of the certificates that need to be renewed)
+## Automated Renewal Process
+
+### IPA Environment (Uses LDAPI)
```

Comment:
It's probably worthwhile to explain why LDAPI may be required (esp. for IPA env).

- DS certificate may be expired, and
- We need to `ldappasswd` some accounts, therefore we need confidentiality, therefore we cannot use LDAPS/STARTTLS, therefore LDAPI :)